docs/content/ssr.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Server-side rendering with Primer React
+---
+
+## SSR-safe ID generation
+
+Some primer components generate their own DOM IDs. Those IDs must be unique across the DOM and isomorphic (so that server-side rendering and client-side rendering yield the same ID, avoiding hydration issues). We use [@react-aria/ssr](https://react-spectrum.adobe.com/react-aria/ssr.html) to generate those IDs. In client-only rendering, this doesn't require any additional work. In SSR contexts, you must wrap your application with at least one `<SSRProvider>`:
+
+```
+import {SSRProvider} from '@primer/components';
+
+<SSRProvider>
+  <MyApplication />
+</SSRProvider>
```

Comment:
Having a floating JSX expression might be confusing to some less experienced readers:
```suggestion
function App() {
  return (
    <SSRProvider>
      <MyApplication />
    </SSRProvider>
  )
}
```